doc/nrf/ug_nrf5340.rst

```diff
@@ -496,6 +496,42 @@ Follow these steps to build and program a multi-image build to the nRF5340 appli
 
 #. Program the application sample using :guilabel:`Target` > :guilabel:`Download zephyr/merged.hex`.
 
+Using |VSC|
+===========
+
+|vsc_extension_instructions|
+
+You can build and program separate images or combined images using |VSC|.
+
+Separate images
+---------------
+
+To build and program the application core, follow the instructions in `Building an application`_ and use ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as the build target.
+
+To build and program the network core, follow the instructions in `Building an application`_ and use ``nrf5340dk_nrf5340_cpunet`` as the build target.
+
+.. _ug_nrf5340_VSC_multi_image:
+
+Multi-image build
+-----------------
+
+If you are working with Bluetooth LE, Thread, Zigbee, or Matter samples, the network core sample is built as a child image when you build the application core image (see :ref:`ug_nrf5340_multi_image` above).
+
+Complete the following steps to build and program a multi-image build to the nRF5340 application core and network core:
+
+.. include:: /includes/vsc_build_and_run_nrf53.txt
+
+6. Select the ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as the target board and ``prj.conf`` as the configuration.
```

Comment:
```suggestion
6. Select the ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as the target board.
```

Whether you use prj.conf or not is sample specific. The extension will select an appropriate conf file by default.